sparkle/src/components/ConversationMessageActions.tsx

```diff
@@ -24,20 +24,21 @@ import { cn } from "@sparkle/lib/utils";
 type ConversationMessageActionsProps = {
   buttons?: React.ReactElement<typeof Button>[];
   messageEmoji?: ConversationMessageEmojiSelectorProps;
-  messageThumb?: ConversationMessageThumbSelectorProps;
+  messageFeedback?: ConversationMessageFeedbackSelectorProps;
 };
 
 export function ConversationMessageActions({
   buttons = [],
   messageEmoji,
-  messageThumb,
+  messageFeedback,
 }: ConversationMessageActionsProps) {
-  if (messageThumb) {
+  if (messageFeedback) {
     buttons.push(
-      <ConversationMessageThumbsSelector
+      <ConversationMessageFeedbackSelector
         key="thumbs-selector"
-        onSubmitThumb={messageThumb.onSubmitThumb}
-        isSubmittingThumb={messageThumb.isSubmittingThumb}
+        feedback={messageFeedback.feedback}
+        onSubmitThumb={messageFeedback.onSubmitThumb}
+        isSubmittingThumb={messageFeedback.isSubmittingThumb}
       />
     );
```

Comment:
It's not directly related to your PR (it's actually the code I moved from front to here) but this code is not ok and it give very strange results in storybook (surprising it works in the app) - new buttons are added on every render - and it's modifying a prop, which is also not always a good idea. Couldn't we just export the 2 components ConversationMessageThumbsSelector and ConversationMessageEmojiSelector and let the user pass the buttons he needs ?

Reply:
Agreed on storybook behavior I found it weird too.
Not sure what you mean by "let the user pass the buttons he needs"?

Reply:
Moved the feedbackSelector out of ConversationMessageActions - revamping the rest should belong to another cleanup PR

Reply:
do something like 
```
<ConversationMessage messages={[
  <ConversationMessageEmojiSelector ../>,
  <ConversationMessageFeedbackSelector ../>
]} />
```